styles/picrew/catppuccin.user.css

```diff
@@ -0,0 +1,1215 @@
+      background: @mantle !important;
+      border-color: @overlay0 !important;
+    }
+    .st-Terms_Cookie,
+    .st-Terms_CookieTitle {
+      color: @text !important;
+    }
+    .st-Terms_Footer {
+      background-color: @mantle !important;
+      color: @subtext1 !important;
+    }
+    .st-Terms_AgreeBtn {
+      background: @surface1 !important;
+      border-color: @surface1 !important;
+      box-shadow: 0 4px 0 @surface2 !important;
+      color: @text !important;
+    }
+    .st-Terms_AgreeBtn:hover {
+      background: @surface2 !important;
+      border-color: @surface2 !important;
+      box-shadow: 0 4px 0 @surface2 !important;
+      color: @text !important;
+    }
+    .popup {
+      background-color: @base !important;
+      border-color: @accent-color !important;      
+    }
+    .popup-header {
+      background-color: @mantle !important;
+    }
+    .popup-body .content a {
+      color: @accent-color !important;
+    }
+    .bellicon {
+      color: @accent-color !important;
+      border-color: @accent-color !important;
+    }
+    .close-btn {
+      border-color: @overlay0 !important;
+      color: @subtext1 !important;
+    }
+    .st-Footer_Wrapper {
+      background-color: @mantle !important;
+      color: @text !important;
+    }
+    .st-Footer_Links_bottom {
+      background: @base !important;
+      color: @subtext1 !important;
+    }
+    .st-Footer .st-Footer_Links div span.st-Header_Logo {
+      filter: none !important;
+    }
+    .sitetop-Twitter,
+    .sitetop-Twitter_Title {
+      background-color: @mantle !important;
+      border-bottom-color: @overlay1 !important;
+    }
+    .item {
+      border-bottom-color: @surface2 !important;
+    }
+    /* login screen */
+    .page-container .panel {
+      background: @mantle !important;
+      border-radius: 6px !important;
+    }
+    .page-container .page-header {
+      color: @text !important;
+    }
+    .login-container_sns .c_text_right {
+      color: @subtext0 !important;
+    }
+    /* "Forgot password?" link (login screen) */
+    .login-container .login-form-container .c_text_right a {
+      color: @accent-color !important;
+    }
+    .login-container hr.sw_hr01 {
+      border-color: @surface1 !important;
+    }
+    .h100 {
+      background: @base !important;
+      color: @text !important;
+    }
+    /* email/password input (login screen) */
+    .login-container input:not([type="checkbox"], [type="radio"]) {
+      background: @base !important;
+      border-color: @surface2 !important;
+      border-radius: 1px !important;
+      color: @text !important;
+    }
+    .login-container input::placeholder {
+      color: @subtext0 !important;
+    }
+    .login-form-container .login-form input:focus {
+      box-shadow: 0 0 0 1px @accent-color !important;      
+    }
+    .pwBox .btn-default {
+      color: @subtext0 !important;
+    }
+    /* search page */
+    fieldset {
+      background: @base !important;
+    }
```

Comment:
By your comment I'm assuming this is a `fieldset` on the search page that looks good with this background color. However it causes some unfortunate theming on the login page - could we use a more specific selector here? Maybe there is a parent element that we can prefix `fieldset` with to only target it on the search page?

![Screenshot 2024-03-17 at 17 29 33 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/872bab9f-bb76-467a-9a4a-a2dcb2e4f6b6)

Reply:
Yeah this was probably before I understood how to use selectors like that, judging from the line number. Anyway I changed it to target only the search page and you're right, this is way better lol
![image](https://github.com/catppuccin/userstyles/assets/139081117/99be2853-43e3-4c5f-9ce5-3076be2fcbc7)

Reply:
Yay awesome! Nice job.